conversation_manager.py

```
@@ -33,7 +27,7 @@ class ConversationManager:
 
     def __init__(self):
         try:
-            self.encoding = tiktoken.encoding_for_model(MODEL_NAME)
+            self.encoding = tiktoken.encoding_for_model(MODE_NAME)
```

Ruff (0.8.2)

Undefined name `MODE_NAME`

(F821)